docs/admin/install/serving/install-serving-with-yaml.md

```diff
@@ -293,24 +299,23 @@ Follow the procedure for the DNS of your choice:
             helloworld-go   http://helloworld-go.default.example.com   helloworld-go-vqjlf   helloworld-go-vqjlf   True
             ```
 
-    1. Instruct `curl` to connect to the External IP or CNAME defined by the
+    1. Instruct `curl` to connect to the External IP address or CNAME defined by the
       networking layer in section 3 above, and use the `-H "Host:"` command-line
-      option to specify the Knative application's host name. For example, if the
-      networking layer defines your External IP and port to be `http://192.168.39.228:32198` and you wish to access the above `helloworld-go` application, use:
+      option to specify the Knative application's hostname. For example, if the
+      networking layer defines your External IP address and port to be `http://192.168.39.228:32198` and you wish to access the above `helloworld-go` application, run the command:
 
         ```bash
         curl -H "Host: helloworld-go.default.example.com" http://192.168.39.228:32198
         ```
 
         !!! success "Verify the output"
-            In the case of the provided `helloworld-go` sample application, the output
-            should, using the default configuration, be:
+            In the case of the provided `helloworld-go` sample application, when using the default configuration the output is:
 
             ```
             Hello Go Sample v1!
             ```
 
-    Refer to the "Real DNS" method for a permanent solution.
+    See the Real DNS method for a permanent solution.
```

Comment:
May also be good in a `!!! tip` (carrying on the theme from last review comment) "By the way, if you're doing thing X, you may want to do thing Y or do thing Z instead of thing X"